src/components/OpprettPlanPage/OppsummeringSteg/OppsummeringSteg.tsx

```diff
@@ -0,0 +1,51 @@
+import { BodyLong, Heading } from "@navikt/ds-react";
+import { oppfolgingsplanFormDefaultValues } from "../FyllUtPlanSteg/form/form-options";
+import { withForm } from "../FyllUtPlanSteg/form/hooks/form";
+import NarDuFerdigstillerPlanenAlert from "./NarDuFerdigstillerPlanenAlert";
+import OppsummeringButtons from "./OppsummeringButtons";
+import PlanFormSummary from "./PlanFormSummary";
+
+interface Props {
+  isPendingFerdigstillPlan: boolean;
+  onGoBack: () => void;
+  onFerdigstillPlanClick: () => void;
+}
+
+const OppsummeringSteg = withForm({
+  defaultValues: oppfolgingsplanFormDefaultValues,
+  props: {} as Props,
+  render: ({
+    form,
+    isPendingFerdigstillPlan,
+    onGoBack,
+    onFerdigstillPlanClick,
+  }) => {
+    return (
+      <section>
```

Comment:
Hadde det gitt mening å bruke vstack her? Her blir det en kombo av ulike måter å space på, f.eks "spacing" for bodylong, og custom margin for alerten inne i selve komponenten. For min del bør "spacing"-definisjon helst kun ligge i container, slik at det blir åpenbart hva som styrer hvilke spacinger, og gjerne at det blir likt